source/documentation/runbooks/013-custom-search.html.md.erb

```diff
@@ -0,0 +1,19 @@
+---
+owner_slack: "#data-catalogue"
+title: Custom Search Config
+last_reviewed_on: 2024-10-01
+review_in: 3 months
+---
+# <%= current_page.data.title %>
+
+Recent changes to the Datahub search behaviour have highlighted the need for control to ensure we retain the ability to provide an optimal search experience.
```

Comment:
```suggestion
Recent changes to the Datahub search behaviour have highlighted the need for control to ensure we retain the ability to provide an optimal search experience.
```
We are concerned about quoted searches and searches with underscores specifically right? That was the common use case in search for us we noticed a change on